Contest_API.md

```diff
@@ -811,7 +811,7 @@ Properties of judgement type objects:
 | :------ | :-------- | :----------
 | id      | ID        | Identifier of the judgement type, a 2-3 letter capitalized shorthand, see table below.
 | name    | string    | Name of the judgement. (might not match table below, e.g. if localized).
-| penalty | boolean ? | Whether this judgement causes penalty time. Required iff contest:penalty\_time is present.
+| penalty | boolean   | Whether this judgement causes penalty time. Required iff contest:penalty\_time is present.
```

Comment:
This is directly tied to the `contest.penalty_time` property. If that is present (and then presumed non-zero), then *each* judgement type should really have this defined for clients to be able to do anything sensible with this data. So I see no point in keeping this nullable.